<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductCreationForm</class>
 <widget class="QDialog" name="ProductCreationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>685</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>685</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Créer un Nouvel Article</string>
  </property>
  <widget class="QGroupBox" name="gbox_parent">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>51</y>
     <width>631</width>
     <height>97</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lbl_parent_product">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>29</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Produit Mère :</string>
    </property>
    <property name="buddy">
     <cstring>cbox_med</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox_article">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>29</y>
      <width>497</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_family">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>57</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Famille Associée:</string>
    </property>
    <property name="buddy">
     <cstring>cbox_family</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox_family">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>57</y>
      <width>476</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_new">
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>57</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/sales/add.png</normaloff>:/sales/add.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbox_inventory">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Inventorier l'Article</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>25</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dernier Inventaire :</string>
    </property>
    <property name="buddy">
     <cstring>last_inventory</cstring>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="last_inventory">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>25</y>
      <width>130</width>
      <height>23</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy HH:mm</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbox_phys_stock">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Stock de produit obtenu  après le nouvel inventaire</string>
    </property>
    <property name="title">
     <string>Stock Physique Actuel</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="ledit_phys_gros">
       <property name="toolTip">
        <string>Entrer la qunatité gros obtenue après comptage physique</string>
       </property>
       <property name="placeholderText">
        <string>Qunatité (Gros)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledit_phys_details">
       <property name="toolTip">
        <string>Entrer la qunatité déconditionnée obtenue après comptage physique</string>
       </property>
       <property name="placeholderText">
        <string>Quantité (Détails)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="ledit_phys_added_gros">
       <property name="toolTip">
        <string>Entrer la qunatité physique gros à ajouter</string>
       </property>
       <property name="placeholderText">
        <string>Qté Ajoutée (Gros)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledit_phys_added_details">
       <property name="toolTip">
        <string>Entrer la qunatité physique déconditionnée à ajouter</string>
       </property>
       <property name="placeholderText">
        <string>Qté Ajoutée (Détails)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="justification">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Indiquer les raisons de la difference entre les quantités physiques et les quantités théoriques obtenues lors du decompte</string>
    </property>
    <property name="placeholderText">
     <string>Indiquer les raisons de la difference entre les quantités physiques et les quantités théoriques obtenues lors du decompte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Justifier de la différence Physique/Théorique</string>
    </property>
    <property name="buddy">
     <cstring>justification</cstring>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbox_theo_stock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Stock de produit indiqué par le systeme</string>
    </property>
    <property name="title">
     <string>Stock Théorique Actuel</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLineEdit" name="ledit_theo_gros">
       <property name="toolTip">
        <string>Quantité (Gros)</string>
       </property>
       <property name="placeholderText">
        <string>Qunatité (Gros)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledit_theo_details">
       <property name="toolTip">
        <string>Quantité (Détails)</string>
       </property>
       <property name="placeholderText">
        <string>Quantité (Détails)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbox_article">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Article</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Code Barre: </string>
    </property>
    <property name="buddy">
     <cstring>ledit_bar_code</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Désignation :</string>
    </property>
    <property name="buddy">
     <cstring>ledit_designation</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_bar_code">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>29</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Référence:</string>
    </property>
    <property name="buddy">
     <cstring>ledit_reference</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_reference">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_designation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbox_price">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lbl_purchase_price">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>35</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix d'Achat :</string>
     </property>
     <property name="buddy">
      <cstring>ledit_puchase_price</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_puchase_price">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_sale_price">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>65</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix de Vente :</string>
     </property>
     <property name="buddy">
      <cstring>ledit_sale_price</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_sale_price">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbox_vta">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>T.V.A</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vta">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>55</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>22</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_tva_percentage">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>110</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>% </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbox_reduction">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Remise</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="reduction">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>55</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>22</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_reduction_percentage">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>110</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>% </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_colise_mere">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Quantité par unité de conditionnement</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_colise_mere">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>35</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Qté Colisée Mère</string>
     </property>
     <property name="buddy">
      <cstring>ledit_colise_mere</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_colise_derive">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Quantité par unité de conditionnement pour le produit derive</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_colise_derive">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>65</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Qté Colisée:</string>
     </property>
     <property name="buddy">
      <cstring>ledit_colise_derive</cstring>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbtn_save">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>650</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enrégister l'article et Passer a la création d'un nouvel article</string>
   </property>
   <property name="text">
    <string>Enregister</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/product/validate.png</normaloff>:/product/validate.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_quit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>650</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Quiter la fenettre sans rien enrégister</string>
   </property>
   <property name="text">
    <string>Quiter</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/sales/cancel.png</normaloff>:/sales/cancel.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_clear">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>650</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Effacer le contenu du formulaire et recommencer pour un nouvel article</string>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/product/clear.png</normaloff>:/product/clear.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbox_prod_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Types de Produits</string>
   </property>
   <widget class="QRadioButton" name="rbtn_medicament">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Médicaments</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_article">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Autres Articles</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cbox_med">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>80</y>
     <width>497</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
